composer.json

```diff
@@ -1,18 +1,22 @@
 {
-    "name": "google/proto-client-php",
+    "name": "google/proto-client",
```

Comment:
No, these don't need to be consistent. For example the google-cloud-php repo has a package name `google/cloud`: https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/composer.json#L2

We want to make this change because (1) having the `-php` suffix is unnecessary, and isn't what other languages do, and (2) this is major breaking change, so it is a good time to do it.